client/stylesheets/_toCat.scss

```diff
@@ -4346,12 +4349,55 @@ On click the images will be displayed at normal size to complete the effect
     margin-left: 5px;
     vertical-align: middle;
     color: #fff !important;
+} */
+
+.sortClicker i {
+	margin-left: 8px;
+	margin-bottom: -2px;
+	color: #fff !important;
+}
+
+.sortClick {
+	float: right !important;
+    margin-right: 30px;
+    font-size: 0.8rem;
+}
+
+.sortItems {
+	margin-top: -2px;
+}
+
+.sortItems li {
+	width: 100%;
+}
+
+.sortClick ul {position:relative;font-weight:400;}
+.sortClick ul li {position:relative;list-style:none;cursor:pointer;}
+.sortClick ul li ul {position:absolute;left:0;right:0;border: 2px solid #04b2ec;}
+.sortClick ul .sortClicker {position:relative;background:#00bfff;color:#FFF;}
+.sortClick ul .sortClicker:hover,.click-nav ul .active {background:#04B2EC !important;}
```

Comment:
Color literals like `#04B2EC` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color `#04B2EC` should be written as `#04b2ec`<br>!important should not be used<br>Selector should have depth of applicability no greater than 2, but was 3<br>Selector `sortClick` should be written in lowercase with hyphens<br>Selector `sortClicker` should be written in lowercase with hyphens<br>Each selector in a comma sequence should be on its own line<br>Colon after property should be followed by one space